gpt4all-chat/flatpak-manifest/io.gpt4all.gpt4all.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>io.gpt4all.gpt4all</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>MIT License</project_license>
+    <name>GPT4ALL</name>
+    <summary>Open-source assistant-style large language models that run locally on your CPU</summary>
+    <description>
+        <p>Cross platform Qt based GUI for GPT4All versions with GPT-J as the base model.</p>
+        <ul>
+            <li>Fast CPU and GPU based inference using ggml for GPT-J and LLaMA based models</li>
```

Comment:
model support is a bit weird right now - latest versions do not support models that were requiring the old llama.cpp forks (gptj, mpt), and only llama is currently supported with Vulkan - we're currently working on updating our llama.cpp to a post-GGUF version before re-adding models so that we can unify all the file handling logic and ideally match file formats with any other ggml implementations of those models (that have also updated to GGUF)

sideloading was always tricky for non-llama ggml models as there's not a single "ggml format" for many model types that were implemented outside llama.cpp as different implementations wound up making different decisions